README.md

```diff
@@ -26,6 +26,16 @@ Update your project's unit testing script in package.json to execute `lwc-jest`:
 
 Alternatively, you can globally install the package and run directly from the command line.
 
+## Updating .forceignore
+
+After adding Jest tests, you'll get errors trying to push your local files with a scratch org because the `__tests__` directory will not be recognized. To ignore these test files, add this entry to your `.forceignore` file:
+
```

Comment:
After adding Jest tests, pushing your local files to a scratch org causes errors because the `__tests__` directory isn't recognized.